src/components/container/ExternalAccountsLoadingIndicator/ExternalAccountsLoadingIndicator.tsx

```diff
@@ -57,6 +57,10 @@ export default function (props: ExternalAccountsLoadingIndicatorProps) {
     }
 
     function initialize() {
+        if (props.previewState) {
+            setIsWeb(true);
+            return;
```

Comment:
Should this call `refresh()` before returning?

Reply:
Yep, totally should...I just realized that myself